webui/react/src/hooks/useModal/UserSettings/useModalCreateUser.tsx

```diff
@@ -196,9 +230,19 @@ const useModalCreateUser = ({ groups, onClose, user }: ModalProps): ModalHooks =
       await form.validateFields();
 
       const formData = form.getFieldsValue();
+
+      const newRoles: Set<number> = new Set(formData.roles);
+      // TODO user should not be able to set roles before they load.
```

Comment:
Do we have other modals with a loading state?

Reply:
The `JupyterLabFullConfig` form in`useModalJupyterLab` shows an example loading state while we wait to load a config file. 

You can see it briefly if you hit this `Show Full Config` button.

![Screen Shot 2022-10-14 at 10 47 58 AM](https://user-images.githubusercontent.com/103522725/195888030-d90e3e2e-fe08-444c-a3dd-e3f28c3f66da.png)

Reply:
Handled loading along these lines.